File: $_algotsu-kaizen/problemo-set/find-minimum-operations-to-make-all-elements-divisible-by-three.c
int minimumOperations(int* nums, int numsSize) {
    int result = 0x0;
    for (int i = 0x0; i < numsSize; ++i) (nums[i] % 0x3) && ++result;
    return result;
}
// 0 - 0
// 1 - 1
// 2 + 1
// 3 - 0
// 4 - 1
// 5 + 1
// 6 - 0

File: $_algotsu-kaizen/problemo-set/find-minimum-operations-to-make-all-elements-divisible-by-three.rs
impl Solution {
    pub fn minimum_operations(nums: Vec<i32>) -> i32 {
        let mut result = 0x0;

        for n in nums {
            if n % 0x3 != 0x0 {
                result += 0x1;
            }
        }

        result
    }
}
